### PR TITLE
Only do a single blur when inertifying subtrees.

### PR DESCRIPTION
**Pull request by alice**

### CODE REVIEW

File: inert.js [hunk 1]
```diff
@@ -110,6 +110,24 @@ class InertRoot {
    */
   _makeSubtreeUnfocusable(startNode) {
     composedTreeWalk(startNode, (node) => { this._visitNode(node); });
+
+    let activeElement = document.activeElement;
+    if (!document.contains(startNode)) {
```
Comment (robdodson) [inline inert.js]: do you know how expensive it is to call this? I haven't worked with `contains` before

Comment (alice) [inline inert.js]: I can't imagine it being significantly more or less expensive than a plain treewalk.

Comment (alice) [inline inert.js]: Yep it's precisely a walk up the ancestor chain (in Blink at least): https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/dom/Node.cpp?sq=package:chromium&dr=CSs&rcl=1471252828&l=766

**Comment by robdodson** [comment]
LGTM. As you said, we should probably look into how iframes would be supported down the line.

**Pull request merged**